urdf/realsense-RS200.macro.xacro

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" ?>
+<!--Develped by Daniel Ordonez 22.05.2018 - daniels.ordonez@gmail.com   
+	INFORMATION:
+		This xacro file a URDF representation of the intel real sense RS200 with the 
+        virtual links representing the position of:
+            * The RGB color camera
+            * Infrared 1 camera
+            * Infrared 2 camera
+            * Virtual depth coordinate frame
+		Configured to work with Gazebo if desired.
+-->
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+<xacro:macro name="realsense-rs200">
+
+	<!-- Camera link -->
+	<link name="rs200_camera">
+		<visual>
+			<origin rpy="0 0 0" xyz="0 0 0"/>
+			<geometry>
+				<mesh filename="package://realsense_gazebo_plugin/models/realsense_camera/meshes/realsense.dae"/>
+			</geometry>
+		</visual>
+		<collision>
+			<geometry>
+          		<box size="0.0078 0.130 0.0192"/>
+        	</geometry>
+		</collision>
+		<inertial>
+        	<mass value="0.0615752"/>
+			<origin rpy="0 0 0" xyz="0 0 0"/>
+			<inertia ixx="9.108e-05"
+					 ixy="0"
+				     ixz="0"
+				     iyy="2.51e-06"
+				     iyz="0"
+				     izz="8.931e-05"
+			/>
+      	</inertial>
+	</link>
+
+	<!-- Virtual links representing the cameras positons and orientations-->
+	<link name="color" />
+	<link name="depth" />
+	<link name="ired1" />
+	<link name="ired2" />
+
+	<!-- Joints positioning the virtual links with respect to the camera main link-->
+	<joint name="color_joint" type="fixed">
+    	<parent link="rs200_camera" />
+    	<child link="color" />
+		<!--<origin xyz="0 -0.046 0.004" rpy="0 0 0"/>-->   
+		<!-- The default position is change since in Rviz the cloud depth axis is Z not X-->
+		<origin xyz="0 -0.046 0.004" rpy="${pi/2} ${pi} ${pi/2}"/>
```

Comment:
In Rviz the depth axis for the camera is the `Z` axis , that is why I create this rotations, to the `color` frame position, since it seems to be the only one referenced as the PointCloud frame.